lms/djangoapps/course_api/fields.py

```diff
@@ -0,0 +1,26 @@
+""" Custom fields for Django REST Framework. """
+
+from __future__ import unicode_literals
+
+from rest_framework import fields
+
+
+class AbsoluteURLField(fields.URLField):
```

Comment:
Moved. At some point we should decide which of this custom code is general enough to move to https://github.com/edx/edx-drf-extensions.